.vib/jaeger/cypress/cypress.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Ping @joancafom so he can review the VIB integration part

Reply:
He is already aware!